chain/internal/tendermint/tendermint_node.go

```diff
@@ -192,51 +187,56 @@ func (tn *TendermintNode) CreateNodeContainer(ctx context.Context, additionalFla
 	cmd = append(cmd, additionalFlags...)
 	fmt.Printf("{%s} -> '%s'\n", tn.Name(), strings.Join(cmd, " "))
 
-	cont, err := tn.Pool.Client.CreateContainer(docker.CreateContainerOptions{
-		Name: tn.Name(),
-		Config: &docker.Config{
-			User:         dockerutil.GetDockerUserString(),
-			Cmd:          cmd,
-			Hostname:     tn.HostName(),
+	cc, err := tn.DockerClient.ContainerCreate(
+		ctx,
+		&container.Config{
+			Image: tn.Image.Repository + ":" + tn.Image.Version,
```

Comment:
This is the third time I've seen something along the lines of, `tn.Image.Repository + ":" + tn.Image.Version`

Should this be it's own function? Something like `DockerImageName() or ImageName()`

Reply:
I've changed these from `fmt.Sprintf("%s:%s", repo, version)` to `repo + ":" + version` because I was touching those lines anyway, and`Sprintf` just isn't necessary for a simple string concatenation.

It may as well be a method off the `Image` type. I'll open a followup PR.